tests/test_utils.py

```diff
@@ -27,3 +31,24 @@ def test_nb_nll_moments(mu, alpha):
     assert np.abs(diff) < 0.2 * deviation
     error_var = np.abs(sample.var() - var_th) / var_th
     assert error_var < 1 / np.sqrt(n_montecarlo)
+
+
+# Test data loading from outside the package (e.g. on RTF)
+@pytest.mark.parametrize("modality", ["raw_counts", "clinical"])
+@mock.patch("pathlib.Path.is_dir")
+def test_rtd_example_data_loading(mocked_function, modality):
+    """
+    Test that load_example_data still works when run from a place where the ``datasets``
+    directory is not accessible, as is when the documentation is built on readthedocs.
+    """
+
+    # Mock the output of is_dir() as False to emulate not having access to the
+    # ``datasets`` directory
+    pathlib.Path.is_dir.return_value = [False, True]
```

Comment:
this should be in parametrize (otherwise the value is [False, True]

Reply:
Indeed... would this work [fa707b9](https://github.com/owkin/PyDESeq2/pull/83/commits/fa707b9da68b88c5326063d40aba2fce96fc2c18)?